pyproject.toml

```diff
@@ -18,10 +18,37 @@ exclude = '''
   )/
   | src/.*/migrations/.*\.py
   | src/olympia/translations/tests/testapp/migrations/.*\.py
-  | src/django-babel/.*\.py
 )
 '''
 
+[tool.ruff]
+exclude = [
+    "src/olympia/wsgi.py",
+    "docs",
+    "node_modules",
+    ".npm",
+    "build*.py",
+    "media",
+    "storage",
+    "logs",
+    "site-static",
+    "static",
+    ".git",
+    "*/migrations/*.py",
+]
```

Comment:
We should also ignore `deps` and `tmp`, but OTOH this should already be taken care of by `.gitignore`, which `ruff` respects by default (a big reason why it's so fast even without thoes directories in the explicit `exclude` list).

So maybe instead simplify this list to only contain things not in `gitignore` already ?

Reply:
https://github.com/mozilla/addons-server/pull/20652/commits/85f7df9e71ff9d03a84c92df200fe7612b975a3d

Seems like black can also respect gitignore too
https://black.readthedocs.io/en/stable/usage_and_configuration/file_collection_and_discovery.html#gitignore

Reply:
https://github.com/mozilla/addons-server/pull/20652/commits/8c42181e7312e1b371b4db990a0b938e830ef796 to reduce black's config too